src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -331,8 +340,8 @@ public void verifyVideoWidth(int widthDesired) {
         content.substring(content.indexOf("px") - 4, content.indexOf("px") - 1)
```

Comment:
I know it's outside of this commit, but the number of magic numbers makes me :disappointed: